test/segment/assertions.go

```diff
@@ -30,7 +30,16 @@ func assertMessageQueued(t *testing.T, cl *segment.Client, username, userID, acc
 				"groupId": accountID,
 			},
 		},
-	}, cl.Client().(*MockClient).Queue[0])
+	}
+	actualMessage := cl.Client().(*MockClient).Queue[0]
+	actualTrackEvent, ok := actualMessage.(analytics.Track)
+	require.True(t, ok)
+	assert.Equal(t, expectedTrackEvent.UserId, actualTrackEvent.UserId)
+	assert.Equal(t, expectedTrackEvent.Event, actualTrackEvent.Event)
+	assert.Equal(t, expectedTrackEvent.Context, actualTrackEvent.Context)
+	assert.Equal(t, expectedTrackEvent.Properties["user_id"], actualTrackEvent.Properties["user_id"])
+	assert.Equal(t, expectedTrackEvent.Properties["account_id"], actualTrackEvent.Properties["account_id"])
+	assert.NotEmpty(t, actualTrackEvent.Properties["epoch_time"]) // not comparing the timestamp in test
```

Comment:
If I understand the issue here is `incompatible timestamp format` and what this PR does is making sure that the timestamp is not empty, but not checking the compatible format. I might be wrong, though, in that case please disregard my comment.

Reply:
@drpaneas The incompatible timestamp is between Amplitude and pendo. This change is to bypass the incompatibility by sending the compatibile timestamp in properties to segment. Hope it helps answer your question.